packages/cli/src/commands/aws/vpc.ts

```diff
@@ -0,0 +1,40 @@
+import { ensureFileExists } from "../../util/ensure-file-exists";
+import { replaceHclValue } from "../../util/replace-hcl-value";
+import { apply } from "../terragrunt/apply";
+import { initModules } from "../terragrunt/init-modules";
+import type { BaseContext } from "clipanion";
+
+export interface VpcSetupInput {
+  context: BaseContext;
+  vpcName: string;
+  vpcDescription: string;
+  verbose?: boolean;
+}
+
+export async function setupVpc(input: VpcSetupInput) {
+  await ensureFileExists({
+    context: input.context,
+    downloadUrl: `https://raw.githubusercontent.com/Panfactum/stack/refs/tags/edge.25-03-04/packages/reference/environments/production/us-east-2/aws_vpc/terragrunt.hcl`,
```

Comment:
I think we want these to be separate files from the reference files as I don't like the idea of our "documentation" potentially breaking live functionality.

Reply:
Perhaps even makes sense to just have these bundled with the CLI.

Reply:
Yeah, the current guide links to this file which is why I used it. I'd want the guide to point to whatever the CLI uses. Maybe we embed them in a collapsible panel in the docs and share the source of truth?

Reply:
Thinking on it more I don't think they should be bundled. Then there's the complexity of ensuring the CLI version has the template you want/need. I like keeping the repository as the source of truth but agree not to use the reference files. We could point the website links to the versioned files in the CLI package instead of the reference package.